tabasco-cli/migrations/2020-03-13-164721_create_devices/up.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE devices (
```

Comment:
Missing license header.

Reply:
Hmm this was copied, but I can add the license to all up.sql in the repo

Reply:
Yeah, I think there's still files dotted around that don't have the license header.  Whoever edits them gets to add it :)